Document Android features supported by Anbox Cloud (#122)
anbox-cloud/discourse-docs: ref/android-features.md

> Anbox Cloud implements support for various features Android has. The following provides an overview which features are currently avilable as part of Anbox Cloud. If a feature is not listed, it is currently not supported.

Reviewer: ```suggestion
Anbox Cloud implements support for various features that Android has. The following table provides an overview of which features are currently available as part of Anbox Cloud. If a feature is not listed, it is currently not supported.
```

Reviewer: "If a feature is not listed, it is currently not supported." - that is inconsistent, because there are also some rows without a check mark in the "Supported" column. Can we remove those rows? Then the sentence makes sense.

Reviewer: > "If a feature is not listed, it is currently not supported." - that is inconsistent, because there are also some rows without a check mark in the "Supported" column. Can we remove those rows? Then the sentence makes sense.

Do we need those rows?

Author: @ru-fu I've updated the wording, can you check if you're fine now?

I want to explicitly call some features out as not being available and not just show a table with all green.

| Feature           | Supported | Notes  |

> |--------------------|:-----------------:|

Reviewer: ```suggestion
|--------------------|:-----------------:|-------------|
```
(Not sure if the number of characters is correct ;) - but we need another column)

Author: The number of characters shouldn't matter, I stopped being precise here :-)

| [OpenGL ES](https://source.android.com/devices/graphics/arch-egl-opengl) <= 3.2   |  ✓  | |

> | [Vulkan](https://source.android.com/devices/graphics/arch-vulkan) |  ✓  | |

Reviewer 2: Maybe we need to put something in the `Notes` column like `this feature is only available for Anbox Cloud deployment where AMD or Intel GPU is installed`

Author: I was thinking about that first but it makes things rather complex. We should keep this more generic and put details to https://anbox-cloud.io/docs/exp/gpu-support

Reviewer 2: I'm fine with that.

> | [Camera](https://source.android.com/devices/camera) |  ✓  | |

Reviewer 2: Picture capture and video shooting are not available for Anbox Cloud deployment where AMD or Intel GPU is installed.

Author: Really? That should work.

Reviewer 2: No, they don't work. As far as I tested in the camera application only the camera preview works. Can you double check that? or could you point me to the MP that you got this fixed recently? I don't recall exactly.

Author: I've tested that on latest images and you're right camera support is still broken for AMD and Intel GPUs. I had this working but looks like we didn't merge the patches.

> | [Sensors](https://source.android.com/devices/sensors) |  ✓  | Currently, the sensors that a client device has are not connected to the Android container when streaming |

Reviewer: ```suggestion
| [Sensors](https://source.android.com/devices/sensors) |  ✓  | Currently, the sensors that a client device has are not connected to the Android container when streaming. |
```

> | [Sensors](https://source.android.com/devices/sensors) |  ✓  | Currently sensors a client device has are currently not connected to the Android container when streaming |

Reviewer: ```suggestion
| [Sensors](https://source.android.com/devices/sensors) |  ✓  | Currently, the sensors that a client device has are not connected to the Android container when streaming |
```

| Location           |  ✓  |          |
| NFC                |      |            |
| [Bluetooth](https://source.android.com/devices/bluetooth) | | |

> | WiFi               |  ✓  | Only simulated WiFi is provided to the Android instance |

Reviewer: ```suggestion
| WiFi               |  ✓  | Only simulated WiFi is provided to the Android instance. |
```

> | [Multi Display](https://source.android.com/devices/tech/display/multi_display) | | |

Reviewer: ```suggestion
| [Multi-Display](https://source.android.com/devices/tech/display/multi_display) | | |
```

> | [Data Use](https://source.android.com/devices/tech/datausage)]| | |

Reviewer: ```suggestion
| [Data use](https://source.android.com/devices/tech/datausage)]| | |
```

> | [Data use](https://source.android.com/devices/tech/datausage)]| | |

Reviewer: ```suggestion
| [Data use](https://source.android.com/devices/tech/datausage)| | |
```

Author: Fixed

> | Telephony / Mobile connectivity | | |

Reviewer: ```suggestion
| Telephony / mobile connectivity | | |
```

> | Hardware accelerated video decoding (H.264) | ✓ | |
> | Hardware accelerated video encoding (H.264) | | |

Reviewer: ```suggestion
| Hardware-accelerated video decoding (H.264) | ✓ | |
| Hardware-accelerated video encoding (H.264) | | |
```

Reviewer: Why do we have two rows for this? Both supported and not supported? ;)

Author: One row for decoding, one for encoding. Encoding is currently not supported but decoding is.

> | [fs-verity](https://source.android.com/security/features/apk-verity) | | |

Reviewer: ```suggestion
| [`fs-verity`](https://source.android.com/security/features/apk-verity) | | |
```

| [Disk encryption](https://source.android.com/security/encryption) | | |
| [Verified Boot](https://source.android.com/security/verifiedboot) | | |

> | [TEE](https://source.android.com/security/trusty) | | |

Reviewer: ```suggestion
| [Trusted Execution Environment (TEE)](https://source.android.com/security/trusty) | | |
```
No need to save space here. ;)
